<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RootTestForm</class>
 <widget class="QWidget" name="RootTestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelAppName">
     <property name="text">
      <string>Identity:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="identityLbl">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&lt;unknown&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Desktop file:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="desktopFileLbl">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&lt;unknown&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Can raise:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="canRaiseLbl">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&lt;unknown&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="raiseBtn">
       <property name="text">
        <string>Raise</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Can quit:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="canQuitLbl">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&lt;unknown&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitBtn">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Fullscreen:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Has tracklist:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="hasTracklistLbl">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&lt;unknown&gt;</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Supported URI schemes:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QListWidget" name="uriSchemesList">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Supported mimetypes:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QListWidget" name="mimetypesList">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="fullscreenLbl">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&lt;unknown&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fullscreenOnBtn">
       <property name="text">
        <string>On</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fullscreenOffBtn">
       <property name="text">
        <string>Off</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Can Set Fullscreen</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="canSetFullscreenLbl">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&lt;unknown&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
